pkg/config/shared.go

```diff
@@ -118,14 +118,20 @@ type HTTP struct {
 
 // Redis represents Redis configuration.
 type Redis struct {
-	Address   string   `name:"address" description:"Address of the Redis server"`
-	Password  string   `name:"password" description:"Password of the Redis server"`
-	Database  int      `name:"database" description:"Redis database to use"`
-	Namespace []string `name:"namespace" description:"Namespace for Redis keys"`
+	Address            string   `name:"address" description:"Address of the Redis server"`
+	Password           string   `name:"password" description:"Password of the Redis server"`
+	Database           int      `name:"database" description:"Redis database to use"`
+	Namespace          []string `name:"namespace" description:"Namespace for Redis keys"`
+	Sentinel           bool     `name:"sentinel" description:"Connect to Redis Sentinel"`
```

Comment:
```suggestion
	EnableFailover     bool     `name:"enable-failover" description:"Enable failover using Redis Sentinel"`
```

Reply:
I have updated the code as per the suggestion